psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -186,6 +197,7 @@ def scf_iterate(self, e_conv=None, d_conv=None):
     Drms = 0.0
     while True:
         self.iteration_ += 1
```

Comment:
As a note the `_` coming *after* the name is a leftover from the C++ days. In Python it is canonical that to proceed with the `_` for private variables `self._iteration`.

Reply:
No, that's Python side AFAIK. It might tie into a more in-depth conversation however on how to handle these hybrid python/C++ classes, so I don't think touching it is correct unless @loriab approves.

Reply:
`self.something_` are indeed python exports of getters/setters to C++ class private member data. So definitely leave `self.iteration_` in particular alone. When py-side and c-side both need to change c-side member data, we get such objects (which you're justified in looking askance at).